packages/zend-form/library/Zend/Form.php

```diff
@@ -495,7 +495,7 @@ public function getPluginLoader($type = null)
      */
     public function addPrefixPath($prefix, $path, $type = null)
     {
-        $type = strtoupper($type);
+        $type = is_string($type) ? strtoupper($type) : $type;
```

Comment:
idem

Reply:
oh, for `addPrefixPath` we have to allow passing `null`, so it's not relevant anymore